pkg/server/store/configmap/cluster.go

```diff
@@ -0,0 +1,107 @@
+package configmap
+
+import (
+	"context"
+	"encoding/json"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/gocrane/crane/pkg/known"
+	"github.com/gocrane/crane/pkg/server/store"
+)
+
+const CraneStoreLabel = "crane-store"
```

Comment:
After discuss with @mfanjie , we suggest to store all cluster infos into one secret in crane-system to support saving some secret infos in cluster.